src/main/kotlin/org/jetbrains/kotlinx/jupyter/htmlUtil.kt

```diff
@@ -0,0 +1,76 @@
+package org.jetbrains.kotlinx.jupyter
+
+import org.jetbrains.kotlinx.jupyter.api.VariableState
+
+// TODO : perhaps, create factory class
+fun generateHTMLVarsReport(variablesMap: Map<String, VariableState>): String {
+    return buildString {
+        this.append(
+            """
+            <!DOCTYPE html>
+            <html>
+            <head>
+            
+            """.trimIndent()
+        )
+        this.append(generateStyleSection())
+        this.append("\n</head>\n<body>\n")
+        this.append("<h2 style=\"text-align:center\">Variables State</h2>\n")
+
+        if (variablesMap.isEmpty()) {
+            this.append("<p>Empty state</p>\n\n")
+            this.append("</body>\n</html>")
+            return this.toString()
+        }
+
+        this.append(generateVarsTable(variablesMap))
+
+        this.append("</body>\n</html>")
+    }
+}
+
+// TODO: text is not aligning in a center
```

Comment:
```suggestion
// TODO: text is not aligned in the center
```

Reply:
By the way, you may spend a bit of time to get rid of this todo, or we may look together into it